TMS.Net07.Homework.DaysOfWeek/Master/Program.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+
+namespace Master
+{
+    class Program
+    {
+        static bool CheckDate(int day, int month, int year)
+        {
+            if (year > 0 && year < 3000)
+            {
+                if (month > 0 && month < 13)
+                {
+                    switch (month)
+                    {
+                        case 1:
```

Comment:
From a code understanding perspective, It would be better to use enum here, something like:
```suggestion
                        case Months.January:
```